### PR TITLE
Update setup and test scripts

### PR DESCRIPTION
**Pull request by darrenleeweber**

### CODE REVIEW

File: lib/moab/utc_time.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+require 'time'
+
+module Moab
+  # Timestamp conversion methods.
+  class Moab::UtcTime
+
+    # @param datetime [Time,String,Nil] The input datetime
+    # @return [void] Convert input datetime to a Time object, or nil if input is empty.
```
Comment (atz) [inline lib/moab/utc_time.rb]: return is *not* void!

Comment (cbeer) [inline lib/moab/utc_time.rb]: This class is just moved from the monkey patches (and.. isn't a monkey patch)

**Comment by atz** [comment]
:+1: to fixing bad habits.

**Comment by darrenleeweber** [comment]
This was merged with potentially significant namespace changes that can impact down-stream projects.  It's not clear that these downstream projects are version pinned to cope with these changes.  All of this should have been done in a feature branch off of develop and, as one of the DLSS devs with more experience on this project, I should be consulted on it before merging.

**Pull request merged**